polyhost/res/overlay_sources/krita/fetch_icons.py

```
@@ -0,0 +1,276 @@
+
+
+def _breeze_svg(name: str) -> tuple[str, bytes]:
+    """Fetch a Breeze icon, trying the action sizes (then apps/) in order and
+    returning the first that resolves. Returns (cat/size, svg_bytes)."""
+    for cat in BREEZE_CATS:
+        for size in BREEZE_SIZES:
+            url = BREEZE.format(cat=cat, size=size, name=name)
+            try:
+                return f"{cat}/{size}", _get(url)
+            except Exception:
+                continue
```

Ruff (0.15.17)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)